MAUI/ImageEditor/save.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Save in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about save support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Save in .NET MAUI ImageEditor (SfImageEditor)
+
+The Image Editor control in .NET MAUI allows you to save the edited image as PNG and JPEG.
+
+To save the modified image, you can use the `Save` method, which accepts parameters such as file name, file type, file path, and image size. The supported file types for saving are PNG and JPEG. You can also save the image by clicking the save button on the toolbar.
+
+{% tabs %}
+
+{% highlight C# tabtitle="C#" %}
+
+              imageEditor.Save(ImageFileType.Png, "D:\\Syncfusion\\Pictures", "Syncfusion");
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Save events
+
+The Image Editor has events when performing the save operation, such as `ImageSaving` and `ImageSaved`.
+
+### ImageSaving
+
+This `ImageSaving` event occurs before saving the image.
+
+`Cancel`: You can control the save functionality by setting the `Cancel` argument to `true`. When `Cancel` is set to `true`, it restricts saving the image to the default location.
```

Comment:
`Cancel`: It restricts the saving of the image to the default location.

Reply:
Modified a content